devops/charts/osdu-common/templates/appgw-ingress.yaml

```diff
@@ -40,3 +40,7 @@ spec:
               serviceName: search-service
               servicePort: 80
             path: /api/search/v2/*
+          - backend:
+              serviceName: {{ .Release.Name }}-schema
```

Comment:
Is there a reason you didn't follow the same pattern as already being used as the "Service Name?"

Please see the pattern in use and continue to use the same pattern.

Reply:
@amaverma - Please do the needful here.

Reply:
fixed in the latest iteration